types/office-js-preview/index.d.ts

```diff
@@ -19919,6 +19919,455 @@ declare namespace Excel {
+        basketball = "Basketball",
+        /**
+        * Beaker icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        beaker = "Beaker",
+        /**
+        * Bird icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        bird = "Bird",
+        /**
+        * Book icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        book = "Book",
+        /**
+        * Bridge icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        bridge = "Bridge",
+        /**
+        * Briefcase icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        briefcase = "Briefcase",
+        /**
+        * Car icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        car = "Car",
+        /**
+        * Cat icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        cat = "Cat",
+        /**
+        * City icon. Displays as multiple tall buildings.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        city = "City",
+        /**
+        * Cloud icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        cloud = "Cloud",
+        /**
+        * Constellation icon. Displays dots in the shape of the Big Dipper, with a bowl and a handle.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        constellation = "Constellation",
+        /**
+        * Dinosaur icon. Displays as a long-necked dinosaur, similar to a Brachiosaurus.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        dinosaur = "Dinosaur",
+        /**
+        * Disaster icon. Displays as a house sinking in a flood.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        disaster = "Disaster",
+        /**
+        * DNA icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        dNA = "DNA",
+        /**
+        * Dog icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        dog = "Dog",
+        /**
+        * Drama icon. Displays as a pair of theatre masks.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        drama = "Drama",
+        /**
+        * First aid icon. Displays as a briefcase with a medical cross symbol.
```

Comment:
This doesn't display as a red cross, right? The Red Cross is extremely eager to file lawsuits about that (this is a weirdly common problem in the gaming world).

Reply:
No, the icons are all monochrome. This cross is black. (Attempted to upload a screenshot but that's not working either).